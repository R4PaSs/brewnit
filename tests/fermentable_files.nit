import dummy_db

with ctx = new DBContext do
	var res = ctx.fermentable_worker.fetch_multiple(ctx, "* FROM fermentables")
	for i in res do print i
end
