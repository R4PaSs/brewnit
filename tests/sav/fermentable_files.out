Grain: Two-Row, potential: 1.038 SG, colour: 2.0 SRM
Grain: Crystal 40, potential: 1.037 SG, colour: 40.0 SRM
Adjunct: Pumpkin, potential: 1.0 SG, colour: 0.0 SRM
Adjunct: Cinnamon, potential: 1.0 SG, colour: 0.0 SRM
Adjunct: Cloves, potential: 1.0 SG, colour: 0.0 SRM
Adjunct: Ginger, potential: 1.0 SG, colour: 0.0 SRM
Adjunct: Nutmeg, potential: 1.0 SG, colour: 0.0 SRM
Grain: Flaked Oats, potential: 1.037 SG, colour: 1.0 SRM
Grain: Flaked Wheat, potential: 1.035 SG, colour: 1.6 SRM
Grain: Wheat Malt, potential: 1.036 SG, colour: 2.0 SRM
Adjunct: Rice Hulls, potential: 1.0 SG, colour: 0.0 SRM
Adjunct: Hibiscus/Berries Tisane, potential: 1.0 SG, colour: 0.0 SRM
Grain: Two Row, potential: 1.037 SG, colour: 1.981 SRM
Extract: Light liquid extract, potential: 1.034 SG, colour: 1.981 SRM
Grain: Crystal 120, potential: 1.034 SG, colour: 120.0 SRM
Grain: Roasted Barley, potential: 1.029 SG, colour: 400.0 SRM
Grain: Chocolate Malt, potential: 1.03 SG, colour: 300.0 SRM
Adjunct: Star Anise, potential: 1.0 SG, colour: 0.0 SRM
